enhancements/sig-architecture/81-addon-lifecycle/kustomize-rollout-examples/02-rollout-strategy/hub-scoped-addon-configuration.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: addon.open-cluster-management.io/v1alpha1
+kind: HubScopedAddOnConfiguration
+metadata:
+  name: standard-config
+  annotations:
+    internal.config.kubernetes.io/needsHashSuffix: enabled
+spec:
```

Comment:
we'd likely need a reference in here too and to handle it you'd want that named reference to have  name substitution with the SHA suffix defined in kustomize so a change to the CR content causes a change to this config, which causes a change the ClusterManagementAddOn